File: 30_RESOURCES/RES_Templates/offer-generator/generate.py
#!/usr/bin/env python3
"""
Dokodu Offer Generator
Przyjmuje plik JSON z danymi oferty, generuje PDF przez WeasyPrint.

Użycie:
  python3 generate.py <offer_data.json> [output.pdf]
  python3 generate.py --test
"""

import sys
import json
import os
import base64
from pathlib import Path
from datetime import datetime, timedelta

SCRIPT_DIR = Path(__file__).parent

# ── Wczytaj dane ──────────────────────────────────────────
if len(sys.argv) < 2:
    print("Użycie: python3 generate.py <offer_data.json> [output.pdf]")
    print("        python3 generate.py --test")
    sys.exit(1)

if sys.argv[1] == '--test':
    today = datetime.today()
    valid = today + timedelta(days=14)
    months_pl = ["stycznia","lutego","marca","kwietnia","maja","czerwca","lipca","sierpnia","września","października","listopada","grudnia"]
    fmt = lambda d: f"{d.day} {months_pl[d.month-1]} {d.year}"
    data = {
        "clientName":    "Firma Przykładowa S.A.",
        "clientSlug":    "FirmaPrzykladowa",
        "date":          fmt(today),
        "validUntil":    fmt(valid),
        "coverSubtitle": "Automatyzacja procesów operacyjnych działu obsługi klienta z wykorzystaniem AI.",
        "execSummary":   "Wasz dział BOK obsługuje miesięcznie ok. 2 000 zapytań, z czego 70% to powtarzalne pytania wymagające ręcznego wyszukiwania odpowiedzi. Proponujemy wdrożenie agenta AI zintegrowanego z Waszym CRM, który automatycznie odpowie na 65% zapytań — szacowana oszczędność: 3 etaty i 40 000 PLN miesięcznie.",
        "problemIntro":  "W trakcie discovery call zidentyfikowaliśmy trzy główne obszary generujące straty operacyjne:",
        "pains": [
            {"icon": "⏱", "title": "Czas odpowiedzi: 4–6 godzin", "desc": "Klienci czekają na prostą informację, którą agent AI udzieliłby w 3 sekundy."},
            {"icon": "🔁", "title": "70% zapytań jest powtarzalnych", "desc": "Konsultanci ręcznie piszą te same odpowiedzi każdego dnia."},
            {"icon": "📉", "title": "CSAT spada przez czas oczekiwania", "desc": "Niezadowolenie klientów rośnie proporcjonalnie do czasu odpowiedzi — ryzyko churn."},
        ],
        "approach": [
            {"title": "Analiza i mapowanie", "desc": "Audyt procesów BOK: kategorie zapytań, źródła danych, systemy (CRM, helpdesk)."},
            {"title": "Budowa agenta AI", "desc": "Wdrożenie agenta n8n zintegrowanego z Waszym CRM i bazą wiedzy."},
            {"title": "Testy i kalibracja", "desc": "Dwutygodniowy okres testów z Waszym teamem na realnych przypadkach."},
            {"title": "Wdrożenie i transfer wiedzy", "desc": "Go-live na produkcji i szkolenie Waszego zespołu."},
        ],
        "optionA": {
            "name": "Start AI",
            "deliverables": ["Agent FAQ — 50 najczęstszych zapytań", "Integracja z helpdeskiem", "Dashboard monitoring", "Szkolenie 2 administratorów", "1 miesiąc wsparcia"],
            "features": ["50 zautomatyzowanych odpowiedzi", "Integracja helpdesk", "Szkolenie zespołu", "1 mies. support"],
            "price": "24 900 PLN",
        },
        "optionB": {
            "name": "AI Full Stack",
            "deliverables": ["Wszystko z Opcji A", "Agent 200+ scenariuszy", "Integracja CRM + helpdesk + e-mail", "Smart routing do konsultantów", "Retainer 6 miesięcy", "Raport ROI co miesiąc"],
            "features": ["200+ scenariuszy", "CRM + helpdesk + e-mail", "Smart routing", "Retainer 6 mies.", "Raport ROI"],
            "price": "54 900 PLN",
        },
        "timeline": [
            {"week": "Tyg. 1–2", "label": "Analiza i mapowanie", "width": 30},
            {"week": "Tyg. 3–5", "label": "Budowa agenta AI", "width": 55},
            {"week": "Tyg. 6–7", "label": "Testy z teamem", "width": 42},
            {"week": "Tyg. 8",   "label": "Go-live", "width": 20},
            {"week": "Mies. 2–7","label": "Retainer (Opcja B)", "width": 80},
        ],
        "roiHeadline": "Zwrot inwestycji w ok. 2 miesiące",
        "roiDetail":   "Przy koszcie 3 etatów BOK (~40 000 PLN/mies.) oszczędność za rok to ok. 480 000 PLN.",
        "whyUs": [
            {"icon": "🏭", "title": "Znamy Waszą branżę", "desc": "Realizowaliśmy podobny projekt dla firmy produkcyjnej — efekty po 6 tygodniach."},
            {"icon": "⚖️", "title": "Tech + Legal w jednym", "desc": "Wdrożenie i compliance AI Act w jednym pakiecie. Alina (COO/Legal) dba o bezpieczeństwo."},
            {"icon": "🔓", "title": "Zero lock-in", "desc": "Cały kod i konfiguracja na Waszej infrastrukturze."},
            {"icon": "🇵🇱", "title": "Polski support", "desc": "Dostępni w języku polskim, w polskich godzinach pracy."},
        ],
        "ctaSteps": [
            "Odpiszcie z potwierdzeniem wybranej opcji (A lub B)",
            "Prześlę umowę w ciągu 24 godzin",
            "Kickoff call — zarezerwujemy termin od razu",
        ],
    }
    output_path = f"Oferta_Test_{datetime.today().strftime('%Y-%m')}.pdf"
else:
    with open(sys.argv[1]) as f:
        data = json.load(f)
    output_path = sys.argv[2] if len(sys.argv) > 2 else \
        f"Oferta_{data.get('clientSlug','Klient')}_{data.get('date','')[:7]}.pdf"

# ── Logo jako base64 ──────────────────────────────────────
logo_path = SCRIPT_DIR / "assets" / "logo_black.avif"
with open(logo_path, "rb") as f:
    logo_b64 = base64.b64encode(f.read()).decode()
logo_data_url = f"data:image/avif;base64,{logo_b64}"

# ── HTML builders ─────────────────────────────────────────
def pain_item(icon, title, desc):
    return f'<div class="pain-item"><span class="pain-title">{title}</span><span class="pain-desc">{desc}</span></div>'

def approach_step(num, title, desc):
    return (
        '<table width="100%" cellspacing="0" cellpadding="0" style="margin-bottom:12px;border-collapse:collapse;">'
        '<tr>'
        '<td width="42" style="vertical-align:top;padding-top:2px;">'
        f'<span style="display:inline-block;width:32px;height:32px;border-radius:16px;background:#0F2137;color:#fff;font-weight:800;font-size:11pt;text-align:center;line-height:32px;">{num}</span>'
        '</td>'
        '<td style="vertical-align:top;padding-top:4px;">'
        f'<span style="display:block;font-weight:700;font-size:11pt;color:#0F2137;margin-bottom:3px;">{title}</span>'
        f'<span style="display:block;font-size:10pt;color:#5A6677;">{desc}</span>'
        '</td>'
        '</tr></table>'
    )

def scope_card(featured, badge, name, deliverables):
    cls = "scope-card featured" if featured else "scope-card"
    items = "".join(f'<div class="scope-item">{d}</div>' for d in deliverables)
    return f'<div class="{cls}"><div class="scope-badge">{badge}</div><h3>{name}</h3>{items}</div>'

def scope_cols(a, b):
    return f'<div class="scope-col">{a}</div><div class="scope-col">{b}</div>'

def timeline_item(week, label, width):
    return (f'<div class="tl-row">'
            f'<div class="tl-week">{week}</div>'
            f'<div class="tl-bar-cell"><div class="tl-bar-bg"><div class="tl-bar-fill" style="width:{width}%"></div></div></div>'
            f'<div class="tl-label">{label}</div>'
            f'</div>')

def price_card(featured, option_label, name, amount, amount_sub, features):
    cls = "price-card featured" if featured else "price-card"
    items = "".join(f'<div class="price-feature">{f}</div>' for f in features)
    return (f'<div class="{cls}">'
            f'<span class="price-option-label">{option_label}</span>'
            f'<span class="price-name">{name}</span>'
            f'<span class="price-amount">{amount}</span>'
            f'<span class="price-vat">{amount_sub}</span>'
            f'<hr class="price-hr">{items}</div>')

def price_cols(a, b):
    return f'<div class="pricing-col">{a}</div><div class="pricing-col">{b}</div>'

def why_item(icon, title, desc):
    # icon_labels: map common icons to short text labels
    icon_map = {
        '🏭': 'BRANŻA', '⚖️': 'LEGAL', '🔓': 'OPEN', '🇵🇱': 'PL',
        '🎯': 'CEL', '💡': 'IDEA', '🚀': 'SPEED', '🔒': 'SEC',
    }
    label = icon_map.get(icon, icon[:2] if icon else '★')
    return (f'<div class="why-card">'
            f'<div style="display:inline-block;background:#0F2137;color:#ffffff;font-size:7pt;font-weight:800;letter-spacing:1px;padding:4px 8px;border-radius:6px;margin-bottom:10px;">{label}</div>'
            f'<span class="why-title">{title}</span>'
            f'<span class="why-desc">{desc}</span>'
            f'</div>')

def why_rows(items):
    html = ''
    for i in range(0, len(items), 2):
        pair = items[i:i+2]
        cells = "".join(f'<div class="why-col">{why_item(**p)}</div>' for p in pair)
        if len(pair) < 2:
            cells += '<div class="why-col"></div>'
        html += f'<div class="why-row">{cells}</div>'
    return html

def cta_step(num, text):
    return (f'<div class="cta-step">'
            f'<div class="cta-num-cell"><span class="cta-num">{num}</span></div>'
            f'<div class="cta-text-cell">{text}</div>'
            f'</div>')

# ── Buduj sekcje ──────────────────────────────────────────
pain_html     = "".join(pain_item(**p) for p in data["pains"])
approach_html = "".join(approach_step(i+1, s["title"], s["desc"]) for i,s in enumerate(data["approach"]))

card_a = scope_card(False, "Opcja A — MVP", data["optionA"]["name"], data["optionA"]["deliverables"])
card_b = scope_card(True, "Opcja B — Pełne rozwiązanie", data["optionB"]["name"], data["optionB"]["deliverables"])
scope_html    = scope_cols(card_a, card_b)

timeline_html = "".join(timeline_item(t["week"], t["label"], t["width"]) for t in data["timeline"])

pc_a = price_card(False, "Opcja A", data["optionA"]["name"], data["optionA"]["price"], "netto + 23% VAT", data["optionA"]["features"])
pc_b = price_card(True,  "Opcja B", data["optionB"]["name"], data["optionB"]["price"], "netto + 23% VAT", data["optionB"]["features"])
pricing_html  = price_cols(pc_a, pc_b)

why_html      = why_rows(data["whyUs"])
cta_html      = "".join(cta_step(i+1, s) for i,s in enumerate(data["ctaSteps"]))

# ── Wczytaj i podmień szablon ─────────────────────────────
template_path = SCRIPT_DIR / "template.html"
html = template_path.read_text(encoding="utf-8")

replacements = {
    "{{LOGO_PATH}}":      logo_data_url,
    "{{CLIENT_NAME}}":    data["clientName"],
    "{{DATE}}":           data["date"],
    "{{VALID_UNTIL}}":    data["validUntil"],
    "{{COVER_SUBTITLE}}": data["coverSubtitle"],
    "{{EXEC_SUMMARY}}":   data["execSummary"],
    "{{PROBLEM_INTRO}}":  data["problemIntro"],
    "{{PAIN_ITEMS}}":     pain_html,
    "{{APPROACH_STEPS}}": approach_html,
    "{{SCOPE_COLS}}":     scope_html,
    "{{TIMELINE_ITEMS}}": timeline_html,
    "{{PRICING_COLS}}":   pricing_html,
    "{{ROI_HEADLINE}}":   data["roiHeadline"],
    "{{ROI_DETAIL}}":     data["roiDetail"],
    "{{WHY_ROWS}}":       why_html,
    "{{CTA_STEPS}}":      cta_html,
}

for key, val in replacements.items():
    html = html.replace(key, val or "")

# ── Generuj PDF ───────────────────────────────────────────
from weasyprint import HTML, CSS
from weasyprint.text.fonts import FontConfiguration

font_config = FontConfiguration()
css = CSS(string='''
  @import url('https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@400;500;600;700;800&family=DM+Sans:wght@400;500;600&display=swap');
  @page { margin: 0; size: A4; }
''', font_config=font_config)

print(f"Generuję PDF: {output_path} ...")
HTML(string=html, base_url=str(SCRIPT_DIR)).write_pdf(
    output_path,
    stylesheets=[css],
    font_config=font_config,
    optimize_images=True,
)
print(f"✅ PDF wygenerowany: {output_path}")

File: 30_RESOURCES/RES_Templates/offer-generator/generate.js
#!/usr/bin/env node
/**
 * Dokodu Offer Generator v2
 * Przyjmuje plik JSON z danymi oferty, generuje PDF.
 *
 * Użycie:
 *   node generate.js <offer_data.json> [output.pdf]
 *   node generate.js --test
 */

const { chromium } = require('playwright');
const fs = require('fs');
const path = require('path');

// ── Wczytaj dane ──────────────────────────────────────────
const args = process.argv.slice(2);
let data;

if (args[0] === '--test') {
  data = getTestData();
} else if (args[0]) {
  data = JSON.parse(fs.readFileSync(args[0], 'utf8'));
} else {
  console.error('Użycie: node generate.js <offer_data.json> [output.pdf]');
  console.error('        node generate.js --test');
  process.exit(1);
}

const outputPath = args[1] || path.join(
  process.cwd(),
  `Oferta_${data.clientSlug || 'Klient'}_${data.date || new Date().toISOString().slice(0,7)}.pdf`
);

// ── Zbuduj HTML ───────────────────────────────────────────
const templatePath = path.join(__dirname, 'template.html');
let html = fs.readFileSync(templatePath, 'utf8');

const logoBlackPath = path.join(__dirname, 'assets', 'logo_black.avif');
const logoBlackDataUrl = `data:image/avif;base64,${fs.readFileSync(logoBlackPath).toString('base64')}`;

// Logo white for cover (if exists, else use black with filter)
const logoWhitePath = path.join(__dirname, 'assets', 'logo_white.avif');
let logoWhiteDataUrl = logoBlackDataUrl;
if (fs.existsSync(logoWhitePath)) {
  logoWhiteDataUrl = `data:image/avif;base64,${fs.readFileSync(logoWhitePath).toString('base64')}`;
}

// ── Helpery HTML (matchują klasy w template.html) ─────────

function painItem(title, desc) {
  return `<div class="pain-item"><span class="pain-title">${title}</span><span class="pain-desc">${desc}</span></div>`;
}

function approachStep(num, title, desc) {
  return `<div class="step"><div class="step-num-cell"><span class="step-num">${num}</span></div><div class="step-body-cell"><span class="step-title">${title}</span><span class="step-desc">${desc}</span></div></div>`;
}

function scopeCol(featured, badge, title, deliverables) {
  const cardCls = featured ? 'scope-card featured' : 'scope-card';
  const items = deliverables.map(d => `<div class="scope-item">${d}</div>`).join('');
  return `<div class="scope-col"><div class="${cardCls}"><span class="scope-badge">${badge}</span><h3>${title}</h3>${items}</div></div>`;
}

function timelineItem(week, label, widthPct) {
  return `<div class="tl-row"><span class="tl-week">${week}</span><div class="tl-bar-cell"><div class="tl-bar-bg"><div class="tl-bar-fill" style="width:${widthPct}%"></div></div></div><span class="tl-label">${label}</span></div>`;
}

function priceCol(featured, optionLabel, name, amount, amountSub, features) {
  const cls = featured ? 'price-card featured' : 'price-card';
  const items = features.map(f => `<div class="price-feature">${f}</div>`).join('');
  return `<div class="pricing-col"><div class="${cls}"><span class="price-option-label">${optionLabel}</span><span class="price-name">${name}</span><span class="price-amount">${amount}</span><span class="price-vat">${amountSub}</span><hr class="price-hr">${items}</div></div>`;
}

function whyCard(num, title, desc) {
  return `<div class="why-col"><div class="why-card"><div class="why-icon-box">${num}</div><span class="why-title">${title}</span><span class="why-desc">${desc}</span></div></div>`;
}

function agendaTable(title, rows) {
  const rowsHtml = rows.map(r => {
    if (r.break) return `<tr class="agenda-break"><td colspan="3">${r.label || 'Przerwa'}</td></tr>`;
    return `<tr><td class="agenda-time">${hs(r.time)}</td><td class="agenda-block">${hs(r.block)}</td><td class="agenda-desc">${hs(r.desc)}</td></tr>`;
  }).join('');
  const titleHtml = title ? `<h3>${title}</h3>` : '';
  return `${titleHtml}<table class="agenda-table"><thead><tr><th>Czas</th><th>Blok</th><th>Co robimy</th></tr></thead><tbody>${rowsHtml}</tbody></table>`;
}

// ── Twarde spacje (polskie typografia) ────────────────────
// Zamienia "z ", "w ", "i ", "o ", "u ", "a " na wersje z &nbsp;
// żeby krótkie przyimki nie zostawały na końcu linii
function hardSpaces(text) {
  if (!text) return text;
  return text
    .replace(/(\s)(z|w|i|o|u|a|Z|W|I|O|U|A)\s/g, '$1$2\u00A0')
    .replace(/^(z|w|i|o|u|a|Z|W|I|O|U|A)\s/g, '$1\u00A0');
}

function ctaStep(num, text) {
  return `<div class="cta-step"><div class="cta-num-cell"><span class="cta-num">${num}</span></div><div class="cta-text-cell">${text}</div></div>`;
}

// ── Buduj sekcje z danych (z twardymi spacjami) ──────────
const hs = hardSpaces; // alias

const painHtml = (data.pains || []).map(p => painItem(hs(p.title), hs(p.desc))).join('');
const approachHtml = (data.approach || []).map((s,i) => approachStep(i+1, hs(s.title), hs(s.desc))).join('');

const scopeColsHtml = [
  scopeCol(false, data.optionA.badge || 'Szkolenie 1', hs(data.optionA.name), data.optionA.deliverables.map(hs)),
  scopeCol(true,  data.optionB.badge || 'Szkolenie 2', hs(data.optionB.name), data.optionB.deliverables.map(hs)),
].join('');

const timelineHtml = (data.timeline || []).map(t => timelineItem(t.week, hs(t.label), t.width)).join('');

let pricingColsHtml;
if (data.pricingTable && Array.isArray(data.pricingTable)) {
  const rows = data.pricingTable.map(r => {
    if (r.isNote) {
      return `<tr><td colspan="5" style="background:#F8FAFC;font-style:italic;color:var(--muted);padding:10px 12px;">${hs(r.note)}</td></tr>`;
    }
    return `<tr>
      <td>
        <div class="pc-item">${hs(r.item)}</div>
        ${r.meta ? `<div class="pc-meta">${hs(r.meta)}</div>` : ''}
      </td>
      <td class="pc-market">${hs(r.marketPrice)}</td>
      <td class="pc-vector">${hs(r.vectorPrice)}</td>
      <td class="pc-discount">${hs(r.discount || '')}</td>
    </tr>`;
  }).join('');
  const footNote = data.pricingFootnote ? `<tfoot><tr><td colspan="4">${hs(data.pricingFootnote)}</td></tr></tfoot>` : '';
  pricingColsHtml = `<table class="pricing-comparison">
    <thead>
      <tr>
        <th>Pozycja</th>
        <th style="text-align:right;">Cena standardowa</th>
        <th style="text-align:right;">Cena dla ${data.clientName}</th>
        <th style="text-align:center;">Rabat</th>
      </tr>
    </thead>
    <tbody>${rows}</tbody>
    ${footNote}
  </table>`;
} else {
  pricingColsHtml = [
    priceCol(false, data.optionA.badge || 'Szkolenie 1', hs(data.optionA.name), data.optionA.price, 'netto + 23% VAT', data.optionA.features.map(hs)),
    priceCol(true,  data.optionB.badge || 'Szkolenie 2', hs(data.optionB.name), data.optionB.price, 'netto + 23% VAT', data.optionB.features.map(hs)),
  ].join('');
}

// Why rows — 2x2 grid with numbered icons
const whyItems = data.whyUs || [];
let whyRowsHtml = '';
for (let i = 0; i < whyItems.length; i += 2) {
  whyRowsHtml += '<div class="why-row">';
  whyRowsHtml += whyCard(i+1, hs(whyItems[i].title), hs(whyItems[i].desc));
  if (whyItems[i+1]) {
    whyRowsHtml += whyCard(i+2, hs(whyItems[i+1].title), hs(whyItems[i+1].desc));
  }
  whyRowsHtml += '</div>';
}

const ctaHtml = (data.ctaSteps || []).map((s,i) => ctaStep(i+1, hs(s))).join('');

// Agenda pages (optional — each agenda gets its own page)
let agendaPageHtml = '';
if (data.agendas && data.agendas.length > 0) {
  agendaPageHtml = data.agendas.map((a, idx) => `
<div class="page">
  <div class="page-header">
    <div class="logo-cell"><img src="${logoBlackDataUrl}" alt="Dokodu"></div>
    <div class="client-cell">${data.clientName} — Propozycja ${data.date}</div>
  </div>
  <span class="section-label">Agenda${data.agendas.length > 1 ? ` (${idx+1}/${data.agendas.length})` : ''}</span>
  <h2>${hs(a.title)}</h2>
  ${agendaTable('', a.rows)}
  <div class="page-footer">
    <span class="footer-left">Dokodu Sp. z o.o. | kacper@dokodu.it | dokodu.it</span>
    <span class="footer-right">${4 + idx}</span>
  </div>
</div>`).join('');
}

// ── Podmień placeholdery ──────────────────────────────────
const coverTitle = data.coverTitle || `Automatyzacja AI<br>dla <span class="cover-title-highlight">${data.clientName}</span>`;

const replacements = {
  '{{LOGO_COVER}}':     logoWhiteDataUrl,
  '{{LOGO_PATH}}':      logoBlackDataUrl,
  '{{CLIENT_NAME}}':    data.clientName,
  '{{DATE}}':           data.date,
  '{{VALID_UNTIL}}':    data.validUntil,
  '{{COVER_TITLE}}':    coverTitle,
  '{{COVER_SUBTITLE}}': hs(data.coverSubtitle),
  '{{EXEC_SUMMARY}}':   hs(data.execSummary),
  '{{PROBLEM_INTRO}}':  hs(data.problemIntro),
  '{{PAIN_ITEMS}}':     painHtml,
  '{{APPROACH_STEPS}}': approachHtml,
  '{{SCOPE_COLS}}':     scopeColsHtml,
  '{{AGENDA_PAGE}}':    agendaPageHtml,
  '{{TIMELINE_ITEMS}}': timelineHtml,
  '{{PRICING_COLS}}':   pricingColsHtml,
  '{{PRICING_HEADLINE}}': hs(data.pricingHeadline || 'Dwie opcje dopasowane do Państwa potrzeb'),
  '{{ROI_HEADLINE}}':   data.roiHeadline,
  '{{ROI_DETAIL}}':     data.roiDetail,
  '{{WHY_ROWS}}':       whyRowsHtml,
  '{{CTA_STEPS}}':      ctaHtml,
};

for (const [key, val] of Object.entries(replacements)) {
  html = html.replaceAll(key, val || '');
}

// ── Generuj PDF ───────────────────────────────────────────
(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();

  await page.setContent(html, { waitUntil: 'networkidle' });

  await page.pdf({
    path: outputPath,
    format: 'A4',
    printBackground: true,
    margin: { top: '0', bottom: '0', left: '0', right: '0' },
  });

  await browser.close();
  console.log(`✅ PDF wygenerowany: ${outputPath}`);
})();

// ── Dane testowe ──────────────────────────────────────────
function getTestData() {
  const today = new Date();
  const valid = new Date(today); valid.setDate(valid.getDate() + 14);
  const fmt = d => d.toLocaleDateString('pl-PL', { day: 'numeric', month: 'long', year: 'numeric' });

  return {
    clientName:    'SayFlu',
    clientSlug:    'SayFlu',
    date:          fmt(today),
    validUntil:    fmt(valid),
    coverTitle:    `Szkolenia AI<br>dla <span class="cover-title-highlight">SayFlu</span>`,
    coverSubtitle: 'Promptowanie i automatyzacja dla zespolu agencji kreatywnej.',
    execSummary:   'Testowy executive summary.',
    problemIntro:  'Testowy wstep do problemu:',
    pains: [
      { title: 'Raporty kampanii', desc: 'Kazda kampania = raport.' },
      { title: 'Przeciazony email', desc: 'Watki uciekaja, brak priorytetyzacji.' },
    ],
    approach: [
      { title: 'Szkolenie AI (5h)', desc: 'Prompt engineering, GEMs, maile.' },
      { title: 'Szkolenie n8n (7h)', desc: 'Automatyzacja workflow.' },
    ],
    optionA: {
      badge: 'Szkolenie 1',
      name: 'AI & Promptowanie',
      deliverables: ['5h warsztatu', 'Prompt engineering', 'Google GEMs'],
      features: ['5 godzin', 'Do 10 osob', 'Certyfikat'],
      price: '4 900 PLN',
    },
    optionB: {
      badge: 'Szkolenie 2',
      name: 'Automatyzacja n8n',
      deliverables: ['7h warsztatu', 'n8n od zera', '2-3 workflow'],
      features: ['7 godzin', 'Do 10 osob', 'Nagranie'],
      price: '5 900 PLN',
    },
    timeline: [
      { week: 'Krok 1', label: 'Ankieta', width: 15 },
      { week: 'Szkolenie 1', label: 'AI (5h)', width: 40 },
      { week: 'Szkolenie 2', label: 'n8n (7h)', width: 65 },
    ],
    roiHeadline: 'Pakiet: 9 400 PLN netto',
    roiDetail:   'Oszczednosc 1 400 PLN.',
    whyUs: [
      { title: 'Praktyk', desc: 'Live demo, nie PowerPoint.' },
      { title: 'Znamy agencje', desc: 'Szkolilismy Animex.' },
      { title: 'Szyte na miare', desc: 'Dedykowane pod Was.' },
      { title: 'Po polsku', desc: 'Polski jezyk, polska faktura.' },
    ],
    ctaSteps: [
      'Omowcie oferte wewnetrznie',
      'Wybierzcie szkolenie i termin',
      'Wysle ankiete i agende',
    ],
  };
}
